connector/connect/src/main/scala/org/apache/spark/sql/connect/planner/SparkConnectPlanner.scala

```diff
@@ -334,7 +334,11 @@ class SparkConnectPlanner(session: SparkSession) {
   }
 
   private def transformAlias(alias: proto.Expression.Alias): NamedExpression = {
-    Alias(transformExpression(alias.getExpr), alias.getName)()
+    if (alias.getNameCount == 1) {
+      Alias(transformExpression(alias.getExpr), alias.getName(0))()
+    } else {
+      MultiAlias(transformExpression(alias.getExpr), alias.getNameList.asScala.toSeq)
```

Comment:
Can you make a change in Connect DSL and add a server side test in `SparkConnectProtoSuite`? 

Should be straightforward.

Reply:
I actually expect Scala client will re-use Connect DSL. 

if you take a closer look, Connect DSL is DataFrame like internal API that offers `toProto` logic (equivalent to `plan.py` for python). 

Anything adding to the DSL will not longer help testing coverage but also help Scala client.

Reply:
I understand the concepts of the DSL and the testing (I was there when it was written), but thanks a lot for the detailed explanation.

Reply:
I added a new test relation with a `map` type, added the type converters, test agg function and now it covers this branch as well.